hoot-core/src/main/cpp/hoot/core/algorithms/ExpectationIntersection.h

```diff
@@ -33,9 +33,6 @@
 // Tbs
 #include <tbs/stats/TDistribution.h>
 
-// Tgs
-#include <tgs/SharedPtr.h>
```

Comment:
This is where `boost::shared_ptr` is included in most instances.  I deleted the file and all references to it.